runtime/databricks/automl_runtime/forecast/utils.py

```diff
@@ -20,21 +20,22 @@
 
 
 def generate_cutoffs(df: pd.DataFrame, horizon: int, unit: str,
-                     seasonal_period: int, num_folds: int) -> List[pd.Timestamp]:
```

Comment:
Should we make the default value to "D" to simplify the function calls in other places?

Reply:
I'm not sure if it makes sense to have "D" as the default value, because it's just Prophet always reporting the seasonality values in days. It might be more logical to make the `seasonal_unit` optional, and if it's not set, we use the `unit` from the required field. WDYT?